src/main/java/com/mraof/minestuck/data/MinestuckBlockStateProvider.java

```diff
@@ -17,7 +22,28 @@ public MinestuckBlockStateProvider(PackOutput output, ExistingFileHelper exFileH
 	protected void registerStatesAndModels()
 	{
 		
-		simpleBlock(MSBlocks.UNCARVED_WOOD.get());
+		blockWithItem(MSBlocks.UNCARVED_WOOD.get());
 		
 	}
+	
+	private void blockWithItem(Block block)
+	{
+		simpleBlock(block, new ConfiguredModel(cubeAll(block)));
+		simpleBlockItem(block, cubeAll(block));
+	}
+	
+	public void simpleBlockItem(Block block, ModelFile model)
+	{
+		itemModels().getBuilder(key(block).getPath()).parent(model);
+	}
+	
+	public void simpleBlock(Block block, ConfiguredModel... models)
+	{
+		getVariantBuilder(block).partialState().setModels(models);
+	}
+	
+	private ResourceLocation key(Block block)
+	{
+		return ForgeRegistries.BLOCKS.getKey(block);
+	}
```

Comment:
Looks like there's already identical functions in `BlockStateProvider` for these. There's also `simpleBlockWithItem(Block, ModelFile)` which could be used in `blockWithItem(Block)` (which I also recommend renaming to `simpleBlockWithItem(Block)` so that it is an overload of `simpleBlockWithItem(Block, ModelFile)`)

Reply:
The reason I did it like this was for ease of access when making special block or block item models, but I can change it if you want.

Reply:
I'm not quite seeing your point here.
But yeah, I'd like you to change it.

Reply:
alright, done.